README.md

```diff
@@ -1,306 +1,61 @@
-```
-
-Check pod deployment:
-```yaml
-Name:               samplepod
-Namespace:          default
-Priority:           0
-PriorityClassName:  <none>
-Node:               node01/192.168.66.101
-Start Time:         Thu, 14 Feb 2019 13:36:23 +0200
-Labels:             <none>
-Annotations:        k8s.v1.cni.cncf.io/networks: [{"name":"ovs-conf","namespace":"default","mac":"02:00:00:00:00:02"}]
-                    k8s.v1.cni.cncf.io/networks-status:
-                      [{
-                          "name": "flannel.1",
-                          "ips": [
-                              "10.244.0.6"
-                          ],
-                          "default": true,
-                          "dns": {}
-                      },{
-                          "name": "ovs-conf",
-                          "interface": "net1",
-                          "mac": "02:00:00:00:00:02",
-                          "dns": {}
-                      }]
-                    kubectl.kubernetes.io/last-applied-configuration:
-                      {"apiVersion":"v1","kind":"Pod","metadata":{"annotations":{"k8s.v1.cni.cncf.io/networks":"[{ \"name\": \"ovs-conf\"}]"},"name":"samplepod"...
-....
-```
-
-The networks annotation need to contains now a mac field
-```yaml
-k8s.v1.cni.cncf.io/networks: [{"name":"ovs-conf","namespace":"default","mac":"02:00:00:00:00:02"}]
-```
-
-MAC address can be also set manually by the user using the MAC field in the annotation.
-If the mac is already in used the system will reject it even if the MAC address is outside of the range.
-
-### Modifing webhook certificate handling
-
-Part of the kubemacpool functionality is implemented as a
-[kubernetes mutating admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) to ensure that mac is assigned before vm is created.
-
-Using admission webhooks forces application to handle TLS ca and server
-certificates, at kubemacpool this task is delegated to [kube-admission-webook](https://github.com/qinqon/kube-admission-webhook)
-library, this library manages certificates rotation using a controller-runtime webhook server and a small cert manager.
-
-To customize how this rotation works some knobs from the library has being
-exposed as environment variables at kubemacpool pod:
-
-- `CA_ROTATE_INTERVAL`:  Expiration time for CA certificate, default is one year
-- `CA_OVERLAP_INTERVAL`:  Duration the previous to rotate CA certificate live with the new one, defaults to one year
-- `CERT_ROTATE_INTERVAL`:  Expiration time for server certificates, default is half a year
-
-## Develop
-
-This project uses [kubevirtci](https://github.com/kubevirt/kubevirtci) to
-deploy local cluster.
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+          - macAddress: "02:00:00:00:00:00"
+            masquerade: {}
+            name: default
+...
+      networks:
+      - name: default
+        pod: {}
+...
+```
```

Comment:
Looks like there is extra section ending

Reply:
not sure what you mean. the `...` is there so clarify I only took a small snippet from the vm yaml..

Reply:
aaaah, of course, blind me